### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by alesmit** [arelstone/react-native-email-chip#27]
I wrapped `EmailChipInput` into another component, mainly to set some default values for the props. This helps me use the email input in different parts of the app, without passing props for style customization over and over. Then, I created a simple unit test for my wrapper component, but when I run it I get this error:
```
● Test suite failed to run

    myProject/node_modules/@arelstone/react-native-email-chip/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import EmailChipInput from './src/EmailChipInput'
                                                                                             ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```

I'm using Jest, and test code of my wrapper component looks like this:
```typescript
import React from 'react';
import 'react-native';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MyEmailInput from '@components/MyEmailInput';

describe('MyEmailInput', () => {
  it('renders correctly', () => {
    const component = shallow(<MyEmailInput onSubmit={() => {}} emails={[]} />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
```

I ended up removing this unit test due to its failure. Is there anything you could do to help me address this problem?

**Comment by arelstone** [comment]
@alesmit I've done the same and haven't had this issue. Could you please share the code for MyEmailInput

**Comment by alesmit** [comment]
@arelstone sure. Here you go:
```typescript
import React from 'react';
import { StyleSheet } from 'react-native';
import EmailChipInput from '@arelstone/react-native-email-chip';
import { colors } from '@constants';
import { translate as t } from '@locales';

type Props = {
  emails: string[];
  onSubmit: (emails: string[]) => void;
};

const EmailInput: React.FC<Props> = props => (
  <EmailChipInput
    blurOnSubmit={true}
    onSubmit={props.onSubmit}
    entries={props.emails}
    containerStyle={styles.container}
    chipContainerStyle={styles.chipContainerStyle}
    chipTextStyle={styles.chipTextStyle}
    placeholderTextColor={colors.ayGrey64Color}
    inputContainerStyle={styles.inputContainer}
    inputStyle={styles.input}
    placeholder={t('components.emailInput.placeholder')}
  />
);

const styles = StyleSheet.create({
  container: {
    padding: 8,
    borderRadius: 4,
  },
  inputContainer: {
    width: '100%',
    padding: 0,
  },
  input: {
    paddingVertical: 8,
    paddingHorizontal: 8,
    fontSize: 16,
    color: colors.ayBlackColor,
  },
  chipContainerStyle: {
    borderRadius: 4,
    backgroundColor: colors.ayGrey8Color,
    paddingVertical: 8,
    margin: 8,
  },
  chipTextStyle: {
    color: colors.ayGrey80Color,
  },
});

export default EmailInput;
```

The error occurs on every unit test of components that use `EmailInput`. As a temporary solution, after some research, I am creating a dummy component in a `__mock__` folder instead of removing tests for those components.

```typescript
import React from 'react';
import { View } from 'react-native';

type Props = {
  emails: string[];
  onSubmit: (emails: string[]) => void;
};

const EmailInput: React.FC<Props> = () => <View />;

export default EmailInput;
```

I think these should be enough to reproduce the issue, if not please let me know how I can help.

**Comment by arelstone** [comment]
@alesmit This should be fixed in `v 2.0.0`. If the issue still persists please reopen this issue
